src/lib/components/streams/channel-details.svelte

```diff
@@ -21,6 +22,12 @@
 
     async function manageChannelData(): Promise<void> {
         if (subscriptionStatus === SubscriptionState.Authorized) {
+            if (channel.type === ChannelType.public) {
```

Comment:
Do public channel can be readed also when subscription status if not SubscriptionState.Authorized?

Reply:
only on the /history route for the detail route it is only possible if authorized to have a consistent behavior